### DISCUSSION TITLE
Is there away in v1.2 to assemble docker image from a local folder (as it was in v0.13), not from a model store?

### DISCUSSION THREAD

**Discussion by Ziange** [bentoml/BentoML#4806] [Q&A]
There was a command save_to_dir(), that could save all bento files, model(s) and artifacts into a specific folder. And one could assemble model from that folder. Is there a way to do the same in a newer versions?

**Comment by aarnphm** [comment]
Are you looking for saving the bento? All of the bento can be found under `BENTOML_HOME/bentos`. This will include all required files for you to build the bento.
